Dictionary, publication request, and warnings

saulakravitz opened the pull request.

saulakravitz committed:
added data dictionary

saulakravitz committed:
warnings and publication request

mvonsieb commented:
Download link for DD does not work on the branch. Will the actual DD spreadsheet be added later? Several occasions where CodexRT should be spelled "CodeX RT", consistent with the overview and index pages.

saulakravitz committed:
fix link

saulakravitz commented:
Spreadsheet is there, was just not linked from dictionary.md.... sigh.  Fixed
Updated now.  It is at -- http://build.fhir.org/ig/HL7/codex-radiation-therapy/branches/dictionary/CodexRTDataDictionary-STU1.xlsx

saulakravitz committed:
fix broken link

mvonsieb reviewed (approved):
Spelling should be consistently CodeX RT. Otherwise good to merge.

saulakravitz committed:
consistent CodeX RT
